sunpy/map/header_helper.py

```diff
@@ -361,3 +362,54 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, wavelengt
     'CD2_2':
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
+
+
+def carrington_header(date, observer_coordinate, shape, *, projection_code="CAR"):
```

Comment:
Could we change the name here to be consistent with `make_fitswcs_header`? 
```suggestion
def make_carrington_header(date, observer_coordinate, shape, *, projection_code="CAR"):
```

Reply:
Good shout, done